### ISSUE TITLE
Quelques réflexions, concepts et relations autour de la gouvernance partagée 

### ISSUE THREAD

**Issue by GuillaumeAV** [assemblee-virtuelle/pair#154]
En complément de PAIR, on peut envisager une ontologie dédiée à gouvernance partagée des organisations ... 

Concepts clés présents dans PAIR
- Individu
- Organisation
- Rôle
- Statut
- Tâche
- Document
- Type de document

**Classes manquantes**
- Liées aux décisions et aux réunions
   - Information
   - Action / to-do
   - Décision
- Liées aux organisations
   - Groupe de travail
   - Cercle 
- Liées aux activités 
    Chantier 

**Relations manquantes dans PAIR :**
- Sous relations de "est impliqué dans"  (Agent => Activité)
    - est responsable de
    - est référent de 
    - est contributeur de 
-  Relations entre les rôles / entre les cercles / entre les cercles et les rôles 
    - Brainstormer

**Comment by GuillaumeAV** [comment]
@bouviermullerp @David Faveris vous avez plein d'idées non ?

**Comment by bouviermullerp** [comment]
- Action / to-do
   - Responsable
   - Deadline

- Décision
   - Pourquoi ? Hypothèses, Indicateurs de mesure.

**Comment by tfrancart** [comment]
>    Liées aux décisions et aux réunions
>        Information
>        Action / to-do
>        Décision
>    Liées aux organisations
>        Groupe de travail
>        Cercle
>    Liées aux activités
>        Chantier
>    Liées aux évenements
>        Sprint / itération
>        Sprint review
>        Sprint planning
>        Daily scrum
>        Et autres concepts agiles

Ce ne serait pas à proprement parler des sous-classes mais bien des types dans des vocabulaires contrôlés : types d'activité, types d'agent.

Par contre la notion de Décision n'est pas dans PAIR, et il y aurait peut-être effectivement des choses à creuser dans ce sens. Qu'est-ce qu'une décision ? comment a-t-elle été prise ? par qui ? c'est une sous-classe d'Idea ? quelle était la proposition de départ, et la décision à l'arrivée, etc. ?


> Sous relations de "est impliqué dans" (Agent => Activité)
>
>   est responsable de
>    est référent de
>  est contributeur de

Le mécanisme d'assocation réifiée permet déjà de caractérisé le rôle d'un Acteur dans une Activité, par le biais de la propriété `involvmentRole`. Pas besoin de sous-relations à mon sens.

**Comment by GuillaumeAV** [comment]
Yes en effet, on est déja très bien équipé, et il manque la notion de décision ... 

@LiseGlo est-ce que c'est des sujets qui t'intéressent ?

**Comment by GuillaumeAV** [comment]
Voir #14
